include/pytomlpp/toml_types.hpp

```diff
@@ -48,7 +48,7 @@ namespace pybind11 { namespace detail {
 
             toml::time t;
 
-            if (PyDate_Check(src.ptr())) {
+            if (PyTime_Check(src.ptr())) {
```

Comment:
This would suggest we need better unit testing :)

Reply:
Yeah, it seems for `toml::time` we don't have proper tests. :sweat_smile: